c++/include/orc/Reader.hh

```diff
@@ -245,6 +251,11 @@ namespace orc {
      * What scale should all Hive 0.11 decimals be normalized to?
      */
     int32_t getForcedScaleOnHive11Decimal() const;
+
+    /**
+     * Get search argument for predicate push down
+     */
+    std::shared_ptr<SearchArgument> getSearchArgument() const;
```

Comment:
It would be useful for debugging to get statistics about the number of row groups the reader skipped because of the sargs. Could you add such a function to the interface?

Reply:
Yes, that would be helpful. Will add it once I have received enough feedback and probably after ORC-611 & ORC-612 have been fixed.